frontend/src/app/reducers/accounts-reducer.js

```diff
@@ -73,6 +77,7 @@ function _mapAccount(account, owner, systemName, pools, allBuckets) {
         accessKeys: { accessKey, secretKey },
         allowedIps: allowed_ips,
         externalConnections,
+        deleteRestriction,
```

Comment:
Please call the state field `undeletable` so it will have the same name as in hostPools, cloudResorurce and buckets state (it had the same meaning).